src/Footer/FooterContainer1/GridItem2.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from "react";
```

Comment:
dont make new component which is same.

Use `GridItem1` component with different props to render different results.